src/main/kotlin/com/zepben/evolve/services/network/NetworkServiceComparator.kt

```diff
@@ -988,6 +1022,48 @@ class NetworkServiceComparator @JvmOverloads constructor(
             compareIdReferences(TapChanger::tapChangerControl)
         }
 
+    private fun compareSynchronousMachine(source: SynchronousMachine, target: SynchronousMachine): ObjectDifference<SynchronousMachine> =
+        ObjectDifference(source, target).apply {
+            compareRotatingMachine()
+
+            compareValues(
```

Comment:
todo: should be checking the `ReactiveCapabilityCurve` relationship. You need to check why this has not been detcted by the tests.